packages/elements-next/src/base/variables.ts

```diff
@@ -0,0 +1,45 @@
+import { css } from 'linaria'
+export const variables = css`
```

Comment:
So this file is imported in `global.ts`
and in `global.ts`
```
export const globalStyles = css`
  :global() {	
    ${variables}
	...
	}
`
```
In this case, I think `${variables}` should be a string, not wrapped in `css`
```
export const variables = `
	/* color variables */
	--color-white: #fff
	...
```